### PR TITLE
Register neerav.is-a.dev

### PR DESCRIPTION
**Pull request by BadPotato1007**
Register neerav.is-a.dev with TXT record pointing to vc-domain-verify=neerav.is-a.dev,c15e13a6aa8319e806ec.

### CODE REVIEW

**Comment by BadPotato1007** [comment]
btw, the real site I want to point it to is https://neerav.vercel.app/
I will be using vercel to host it, so might as well add it with a record there

**Comment by BadPotato1007** [comment]
i see, you dont support sites supported on vercel. how can we make this work then?

**Review by iostpa — DISMISSED** [review]
We don't allow vercel.

**Comment by CuteDog5695** [comment]
> i see, you dont support sites supported on vercel. how can we make this work then?

You can try to use Netlify

**Comment by BadPotato1007** [comment]
hey there! migrated to netlify, and also added www.neerav.is-a.dev :)

**Comment by BadPotato1007** [comment]
@CuteDog5695

File: domains/www.neerav.json
Comment (orxngc) [inline domains/www.neerav.json:1]: we do not support www subdomains

File: domains/neerav.json
Comment (orxngc) [inline domains/neerav.json:1]: inaccessible website, please provide a preview

Comment (orxngc) [reply domains/neerav.json:1]: nevermind i just saw the other comment

File: domains/www.neerav.json
Comment (BadPotato1007) [reply domains/www.neerav.json:1]: removed that file

**Comment by BadPotato1007** [comment]
done! lets get this done now!

File: domains/neerav.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        "owner": {
+           "username": "BadPotato1007",
+           "email": "",
+           "discord": "1176835313356767312",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.h5lVmEVgNjSJED4_lCqP0uq8U6PetLRaVdYIsrVbDslLhV0Nmvkdj9AbxxT1TGf3ZGwFSSJe-0zei3AeiQ-opshMYrw_DRlx4f686t-AhNXj9Kmf2-0wQxjsmUvwXNsmNfgwxjXtD5gHv8jLOF1oh_ZV9rbjCmWUejPLX6a0RvRk-hH9ACxziJ3qzdY88SdlXcqA6KqxhD0ZDyWt_NAXk-DIDv-oOoeX-MM9ZW1f89HDXvQPZ3uqOmcdzJtoCBGGAt4nY9C0fkK0YGSp_lXe_r6G7gjhNfwEH_BQ997SpDpibdH3ensKeGf6DOufxaN1FK8L0KCf-XpLXELc9c2qUw.bEd83RTDvTOZ19pDJ6YrTA.hj0EVK9MR1_ygJEtgrhofZrU37pBoHu-zXlTGQSG0DuRAi1VQC3jOP6QrMJZi7zWZbce2Y83elAYzq_I4W0KrshAwfAUlcr6DRTb5Hl4JdU.RFRNhW4hhRsoe9Mw-F-fwg"
+        },
+    
+        "record": {
+            "CNAME": "apex-loadbalancer.netlify.com",
+            "A": "75.2.60.5"
```
Comment (DEV-DIBSTER) [inline domains/neerav.json:11]: Cannot have CNAME records in conjunction with other records.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/neerav.json (hunk 1 above)
Comment (BadPotato1007) [reply domains/neerav.json]: Done

**Comment by BadPotato1007** [comment]
Please check now

**Comment by BadPotato1007** [comment]
Please check now

**Review by DEV-DIBSTER — DISMISSED** [review]

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**